tutor/specs/e2e/student-task.e2e.ts

```diff
@@ -0,0 +1,93 @@
+import faker from 'faker'
+import { visitPage, setRole, setTimeouts } from './helpers'
+
+describe('Student Tasks', () => {
+
+    beforeEach(async () => {
+        await page.setViewportSize({ width: 1200, height: 720 })
+        await setTimeouts()
+        setRole('student')
+    });
+
+    const longFreeResponseAnswer = faker.lorem.words(510);
+
+    it('advances after answering a free-response only question', async () => {
+        await visitPage(page, '/course/1/task/3') // task id 3 is a hardcoded WRM task
+        await page.evaluate(() => {
+            window._MODELS.feature_flags.set('tours', false);
+        })
+        await page.click('.icon-instructions')
+        await expect(page).toHaveSelector('testEl=homework-instructions')
+        await page.click('testEl=value-prop-continue-btn')
+        await expect(page).toHaveSelector('testEl=student-free-response')
+        await page.type('testEl=free-response-box', 'this is a answer answering and fully explaining my reasoning for the question')
+        await page.click('testEl=submit-answer-btn')
+        expect(await page.evaluate(() => document.location.pathname)).toContain('/course/1/task/3/step')
+        
+    })
+
+    it.only('can change and re-submit answers to questions', async () => {
```

Comment:
`only` left in.  Can you investigate https://www.npmjs.com/package/eslint-plugin-no-only-tests and add it to the config if it works?

Reply:
@nathanstitt this is a good idea. i got it running in another branch, but I would like to add a `pre-commit`, `pre-push` hooks to check for any linting issue. What do you think?

Reply:
yeah I have a pre-commit hook locally that calls 'yarn run lint`, that'd be great to enforce for everyone